docs/workforce-management-framework/topics/con-workflow-step.adoc

```diff
@@ -0,0 +1,60 @@
+= Workflow Step
+
+== Step Structure
+Steps are divided into to linked catergories the `module` and the `definition`.
+
+=== Module
+Modules can contain one or more angular directives for rendering steps in the portal and mobile applications. 
+{WFM-RC-NameShort} steps use Angular.js directives to build user interface features as part of workflows.
```

Comment:
Can we get an image for this? ie: the 'step' showing in the "build user interface features"

Reply:
to build user interface features => `to build user interface`. 

@jstaffor - misunderstanding - users will build their user interface here - having image may be confusing IMHO

Reply:
Ahh, I understand now - happy not to have an image. :+1: